### PR TITLE
Feature python demo nlclassifier

### PR DESCRIPTION
**Pull request by VihangaAW**
Added a Python wrapper for NLClassifier

### CODE REVIEW

File: tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+def classify(model_path, text):
+    """Classifies input text into different categories
+
+    Args:
+        model_path: path to model
+        text: input text
+
+    """
+    # Run the detection tool:
+    subprocess.run(["bazel run -c opt  tensorflow_lite_support/examples/task/text/desktop:nl_classifier_demo --  --model_path=" + model_path + "  --text='" + text + "'"], shell=True)
```
Comment (xunkai55) [inline tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py:37]: Please format this file with Google Python style https://google.github.io/styleguide/pyguide.html

1. Lines should be <= chars
2. indent size == 2

Comment (VihangaAW) [reply tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py]: Hi @xunkai55 ! I formatted the code. Could you please have a look. Thanks

File: tensorflow_lite_support/examples/task/text/desktop/python/README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
 # CLI Demos for Python Text Task APIs
 
+A Python wrapper for the C++ Text Task APIs.
```
[the remark this answers is not in the document]
Comment (VihangaAW) [reply tensorflow_lite_support/examples/task/text/desktop/python/README.md:3]: I formatted the code in this file also. Btw I bypassed wrapping long lines when it comes to paths.

**Review by lu-wang-g — APPROVED** [review]
Thanks for submitting the PR again!

File: tensorflow_lite_support/examples/task/text/desktop/python/README.md [hunk 3]
```diff
@@ -1,2 +1,47 @@
...
+
+## Background
+This Python API is based on the C++ Text Task APIs. It uses Python's 
+[subprocess](https://docs.python.org/3/library/subprocess.html)
```
Comment (lu-wang-g) [inline tensorflow_lite_support/examples/task/text/desktop/python/README.md:7]: Tailing spaces by the end of these two lines. Please remove.

File: tensorflow_lite_support/examples/task/text/desktop/python/README.md [hunk 4]
```diff
@@ -1,2 +1,47 @@
...
+[subprocess](https://docs.python.org/3/library/subprocess.html) 
+to call C++ Text Task APIs.
+
+## NLClassifier
+
+#### Prerequisites
+
+You will need:
+
+* a TFLite text classification model with certain format.
```
Comment (lu-wang-g) [inline tensorflow_lite_support/examples/task/text/desktop/python/README.md:16]: Same here for tailing space.

File: tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+flags.mark_flag_as_required('model_path')
+flags.mark_flag_as_required('text')
+
+def classify(model_path, text):
+  """Classifies input text into different categories
+
+  Args:
+      model_path: path to model
+      text: input text
+
```
Comment (lu-wang-g) [inline tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py:33]: Please remove the tailing spaces here.

**Review by lu-wang-g — COMMENTED** [review]
Found a few tailing spaces through the internal code review process. Can you  please try formatting your code again using [YAPF](https://github.com/google/yapf)?

**Comment by VihangaAW** [comment]
Hi @lu-wang-g! I removed the trailing spaces and formatted the code using YAPF. Could you please have a look

**Review by lu-wang-g — APPROVED** [review]
Thanks for the updates!

**Pull request merged**